Shared/ViewModels/SettingsViewModel.swift

```diff
@@ -44,4 +55,26 @@ final class SettingsViewModel: ObservableObject {
 		}.sorted(by: { $0.name < $1.name })
 		self.langs.insert(.auto, at: 0)
 	}
+
+	func sendQuickConnect() {
+		QuickConnectAPI.authorize(code: self.quickConnectCode)
+			.sink(receiveCompletion: { completion in
+				switch completion {
+				case .failure:
+					LogManager.log.debug("Invalid Quick Connect code entered")
```

Comment:
This failure isn't necessarily correct, as a failure here does not indicate an invalid code entered but a failure of the call due to HTTP errors and such. `handleAPIRequestError` also needs to be called to handle the entire completion in all API calls. You can just use `self.handleAPIRequestError(completion: completion)` in this closure, and for future reference the `displayMessage` parameter is to override the default, generic "Error" title and these custom errors need to be localized.